cranelift-codegen/meta/src/gen_inst.rs

```diff
@@ -1058,9 +1080,13 @@ fn gen_builder(
     fmt.indent(|fmt| {
         for inst in instructions.values() {
             gen_inst_builder(inst, &*inst.format, fmt);
+            fmt.empty_line();
         }
-        for format in formats {
+        for (i, format) in formats.iter().enumerate() {
             gen_format_constructor(format, fmt);
+            if i + 1 != formats.len() {
+                fmt.empty_line();
+            }
```

Comment:
If we ever bring in `itertools`: https://docs.rs/itertools/0.8.1/itertools/trait.Itertools.html#method.with_position.